test/libsolidity/GasMeter.cpp

```diff
@@ -129,6 +119,9 @@ BOOST_AUTO_TEST_CASE(simple_contract)
 BOOST_AUTO_TEST_CASE(store_keccak256)
 {
 	char const* sourceCode = R"(
+		// TODO: We should enable v2 again once the yul optimizer is activated.
```

Comment:
Not sure. The only context I have is #5102 and there's no discussion of that in there. My assumption is that abicoder v2 encoder must using some EVM feature that `GasMeter` did not properly account for at that time and the assumption was that it would no longer be a problem when it becomes the default.

I don't think we have an issue for it. Actually, could you create one?